spyder/config/main.py

```diff
@@ -640,4 +644,4 @@
 #    or if you want to *rename* options, then you need to do a MAJOR update in
 #    version, e.g. from 3.0.0 to 4.0.0
 # 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '71.1.0'
+CONF_VERSION = '72.0.0'
```

Comment:
Yeah, I wasn't really sure either. I bumped it because I removed the `spyder_pythonpath` option from the `main` section, although that's a transient one (i.e. it's not saved in `spyder.ini` but in `transient.ini`).

Reply:
That's right; I forgot about that. Yes, I think a major version bump is appropriate.